### PR TITLE
Close changelog for 7.4.0

### PR DESCRIPTION
**Pull request by mikemadden42**

### CODE REVIEW

**Comment by mikemadden42** [comment]
This is the changelog for `7.4.0`.  I still need to copy old changelogs that were released prior to `7.4.0`.  Please let me know if you have questions or comments for the `7.4.0` changelog.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,154 @@
...
+- Fix module-level fields in Kubernetes metricsets. {pull}13433[13433] {pull}13544[13544]
+- Fix panic in Redis Key metricset when collecting information from a removed key. {pull}13426[13426]
+
+*Packetbeat*
+
+- Fix parsing the extended RCODE in the DNS parser. {pull}12805[12805]
+
+*Functionbeat*
+
+- Fix Cloudwatch logs timestamp to use timestamp of the log record instead of when the record was processed {pull}13291[13291]
```
Comment (ph) [inline CHANGELOG.asciidoc]: @mikemadden42 I think we are missing entries for this?

Comment (ph) [reply CHANGELOG.asciidoc]: nevermind its in the "added"

Comment (mikemadden42) [reply CHANGELOG.asciidoc]: Awesome... I tried to keep it in the same section that it originated in.

**Pull request merged**